src/main/java/de/tum/in/www1/artemis/domain/Exercise.java

```diff
@@ -157,7 +157,7 @@
 
     @OneToMany(mappedBy = "exercise", cascade = CascadeType.REMOVE, orphanRemoval = true, fetch = FetchType.LAZY)
     @Cache(usage = CacheConcurrencyStrategy.NONSTRICT_READ_WRITE)
-    @JsonIgnoreProperties("exercise")
+    @JsonIncludeProperties({ "id" })
```

Comment:
@lschlesinger So what does this exactly do?
Previously it read to me like whenever you parse an exercise with posts, don't parse the exercise of a post.

Does it not say whenever you parse an exercise with posts, only parse the ids of the posts?

Reply:
yes it does. we actually do not need to serialize the posts within an exercise or a lecture at all, as we never access the posts through the exercise or lecture holding them. So I am thinking about not serializing posts within a lecture/exercise at all.

Reply:
The "not" in the last sentence was actually too much. Anyway, if you don't need it, I think it makes sense to remove it.

Reply:
yes, I would prefer to reduce it to a minimum to fix the issue that was observed during testing. And I will remove it completely in a separate PR, as I might have to change minor things, e.g. in test code.

Reply:
Sounds good. The main goal here was only to understand what it does anyway.